package.json

```diff
@@ -2,6 +2,7 @@
   "name": "react-formal",
   "version": "0.17.9",
   "description": "Classy HTML form management for React",
+  "jsnext:main": "src/index.js",
```

Comment:
Let me restate then.  `jsnext:main` isn't a standard, it's a pseudo and not one that is adopted.  I don't think we want to bike shed it into `jsnext:main:jsx` or something like that in order to allow an app to use it.  I see the point of wanting to constrain it to ES only, but what ES only app would use this UI-only library in a transpilation pipeline that couldn't handle JSX?  I realize this is a bigger and more complicated conversation, but lacking another meta indicator, why not use this until we have another way?